### ISSUE TITLE
Error when opening the problem list

### ISSUE THREAD

**Issue by qxforever** [IEEE-NITK/CodePal#86]
Yesterday, I recieved `Cannot read property 'length' of undefined`. Is this a bug or a problem with my local settings?

**Comment by 28251536** [comment]
Hello, how can I solve this problem?  It can be used normally in the previous two weeks....

**Comment by adityachirania** [comment]
Hey i have mentioned the possible reasons above..  Are you sure you werent facing any such issue?

**Comment by qxforever** [comment]
> Hey i have mentioned the possible reasons above.. Are you sure you werent facing any such issue?

Acutally I received this message too these days. I'm sure my internet conntection and Codeforces server works well. Is it possible that the Codeforces API was disabled?

**Comment by adityachirania** [comment]
Yeah i remember the problemset api wasnt working a few days ago.. Even some other apis may have stopped working temporarily.. Anyways i think this is a temporary issue and shoukd get
